src/chrome/chromeTargetDiscoveryStrategy.ts

```diff
@@ -14,10 +14,23 @@ import { ITargetDiscoveryStrategy, ITargetFilter, ITarget } from './chromeConnec
 import * as nls from 'vscode-nls';
 const localize = nls.loadMessageBundle();
 
+export class ProtocolSchema {
+    public static unkownVersion(): any {
```

Comment:
Small typo, unknown

Reply:
Return type, ProtocolSchema, and use `new ProtocolSchema` instead of `new this`, I didn't even know `this` would work that way in a static method

Reply:
fixed